cmd/xl-storage.go

```diff
@@ -970,7 +971,8 @@ func (s *xlStorage) WriteMetadata(ctx context.Context, volume, path string, fi F
 			logger.LogIf(ctx, err)
 			return err
 		}
-		buf, err := xlMeta.AppendTo(nil)
+		buf, err := xlMeta.AppendTo(metaDataPool.Get().([]byte)[:0])
```

Comment:
Should we have a dummy function for metadataPoolGet() just to compliment the Put ?

Reply:
Done.